src/cfn/template.yaml

```diff
@@ -12,23 +12,78 @@ Globals:
         MAX_TASKS: !Ref MaxTasksCapacity
         REGION: !Ref AWS::Region
         TO_BUCKET: !Ref WebUIBucket
+        VPC_ID: !Ref VPC
   Api:
     Cors:
       AllowMethods: "'*'"
       AllowHeaders: "'Content-Type,X-Amz-Date,Authorization,X-Api-Key,X-Amz-Security-Token'"
       AllowOrigin: "'*'"
 
 Parameters:
+
   MaxTasksCapacity:
+    Description: The maximum number of Fargate Tasks to be allocated
     Type: Number
     Default: 10
 
+  PrivateSubnetIpBlocks:
+    Description: Comma-delimited list of CIDR blocks for the private subnets
+    Type: CommaDelimitedList
+    Default: "10.0.0.0/22, 10.0.4.0/22, 10.0.8.0/22"
+
+  PublicSubnetIpBlocks:
+    Description: Comma-delimited list of CIDR blocks for the public subnets
+    Type: CommaDelimitedList
+    Default: "10.0.12.0/22, 10.0.16.0/22, 10.0.20.0/22"
+
+  VpcIpBlock:
+    Description: CIDR block for the VPC
+    Type: String
+    Default: 10.0.0.0/16
+    AllowedPattern: ^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(\/(1[6-9]|2[0-8]))$
```

Comment:
Allowed pattern is available only for `Type: String` 🤕

Reply:
Yeah, it's a pain.